_data/retail.yml

```diff
@@ -242,3 +242,10 @@ websites:
       twitter: Zappos
       img: zappos.png
       tfa: No
+
+    - name: Zazzle
+      url: http://www.zazzle.com
+      img: zazzle.png
+      tfa: No
+      twitter: zazzle
+      facebook: zazzle
```

Comment:
Missing new line at the end of the file.
If you can, please use editor that respects `.editorconfig` so that you don't have to do it manually.